ee/modules/110-istio/templates/control-plane/iop.yaml

```diff
@@ -140,7 +140,8 @@ spec:
       proxy:
         image: {{ include "helm_lib_module_image" (list $ (printf "proxyv2%s" $imageSuffix )) }}
         clusterDomain: {{ $.Values.global.discovery.clusterDomain | quote }}
-        resources: {}
+        resources:
+{{ include "helm_lib_resources_management_pod_resources" (list $.Values.istio.sidecar.resourcesManagement "1Mi") | nindent 10 }}
```

Comment:
Why 1Mi for ephemeral storage?

Reply:
By default, the function helm_lib_resources_management_pod_resources adds 0Mi. I felt that this value is not safe.

Reply:
But is 1Mi safe?

Reply:
I think so. the application does not consume disk space. and since this is not a limit, the application will in principle be able to spend more space if needed.